subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1718,6 +1729,51 @@ const uint8_t *ull_adv_pdu_update_addrs(struct ll_adv_set *adv,
 	return adv_addr;
 }
 
+uint8_t ull_adv_time_update(struct ll_adv_set *adv, struct pdu_adv *pdu,
+			    struct pdu_adv *pdu_scan)
+{
+	uint32_t volatile ret_cb;
+	uint32_t ticks_minus;
+	uint32_t ticks_plus;
+	struct lll_adv *lll;
+	uint32_t time_ticks;
+	uint16_t time_us;
+	uint8_t chan_map;
+	uint8_t chan_cnt;
+	uint32_t ret;
+
+	lll = &adv->lll;
+	chan_map = lll->chan_map;
+	chan_cnt = util_ones_count_get(&chan_map, sizeof(chan_map));
+	time_us = adv_time_get(pdu, pdu_scan, chan_cnt, PHY_1M);
+	time_ticks = HAL_TICKER_US_TO_TICKS(time_us);
+	if (adv->ull.ticks_slot > time_ticks) {
+		ticks_minus = adv->ull.ticks_slot - time_ticks;
+		ticks_plus = 0U;
+	} else if (adv->ull.ticks_slot < time_ticks) {
+		ticks_minus = 0U;
+		ticks_plus = time_ticks - adv->ull.ticks_slot;
+	} else {
+		return 0;
```

Comment:
```suggestion
		return BT_HCI_ERR_SUCCESS;
```

To stay consistent with line 1769.